lib/rules/block-opening-brace-newline-after/index.js

```diff
@@ -29,13 +30,22 @@ function rule(expectation, options, context) {
 			possible: ['always', 'always-multi-line', 'never-multi-line'],
 		});
 
-		if (!validOptions) {
+		if (
+			!validOptions ||
+			optionsMatches(options, 'ignore', 'rules') ||
+			optionsMatches(options, 'ignore', 'at-rules')
+		) {
 			return;
 		}
 
 		// Check both kinds of statement: rules and at-rules
-		root.walkRules(check);
-		root.walkAtRules(check);
+		if (!optionsMatches(options, 'ignore', 'rules')) {
+			root.walkRules(check);
+		}
+
+		if (!optionsMatches(options, 'ignore', 'at-rules')) {
```

Comment:
The original issue was to only add `ignore: ["rules"]` to this rule. We should revert this change.